tasks/prime.py

```diff
@@ -230,36 +230,36 @@ def list_moves(self):
         # if this is running locally)
         self.set_default_mto_ids(moves or [])
 
-    @tag(MTO_SERVICE_ITEM, "createMTOServiceItem")
```

Comment:
Test commented out to better see when the update shipment status endpoint was being hit. They will be added back when I take this out of draft